futures-util/src/stream/stream/split.rs

```diff
@@ -142,3 +156,69 @@ impl<T, Item> fmt::Display for ReuniteError<T, Item> {
 
 #[cfg(feature = "std")]
 impl<T: core::any::Any, Item> std::error::Error for ReuniteError<T, Item> {}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::{sink::Sink, stream::StreamExt};
+    use core::marker::PhantomData;
+
+    struct NopStream<Item> {
```

Comment:
This struct is a hack because I couldn't find a type to use without introducing a dependency.  I'm open to suggestions if there's something available that I'm missing.